config/application.rb

```diff
@@ -35,6 +35,8 @@ class Application < Rails::Application
       "#{config.root}/app/docs"
     ]
 
+    config.es_url = ENV.fetch("SEARCHBOX_URL", ENV.fetch("ELASTICSEARCH_URL", "http://localhost:9200"))
```

Comment:
this lets us abstract URL behavior, particularly for our tests. we're already doing this with the searchkick initializer, just exclusively using env vars.